--use_system_libnanomsg=true is broken after the update to 0.9-beta

Because the pkg-config file include directory is now "/usr/local/include" and not "/usr/local/include/nanomsg" using the flag above causes the build to fail since it can't find <nn.h>

Fix would be most likely in binding.gyp to modify the include dir read from the pkg-config file. Also from 0.9-beta onwards the pkg-config file has changed from libnanomsg to nanomsg.

I'll probably get around to sending a PR for this but opening this issue in case anyone else can get to it before me.

Thanks.
Deepak.

I tried doing: 
```
ln -s /usr/local/lib/pkgconfig/nanomsg.pc /usr/local/lib/pkgconfig/libnanomsg.pc
```

But I'm getting:

```
make: Entering directory '/projects/mobx-test/node_modules/nanomsg/build'
  CXX(target) Release/obj.target/node_nanomsg/src/node_nanomsg.o
../src/node_nanomsg.cc:5:16: fatal error: nn.h: No such file or directory
 #include <nn.h>
                ^
compilation terminated.
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -15,10 +15,22 @@
         }],
         ['OS=="linux" and use_system_libnanomsg=="true"', {
           'include_dirs+': [
-            '<!@(pkg-config libnanomsg --cflags-only-I | sed s/-I//g)',
+            '<!@(pkg-config nanomsg --cflags-only-I | sed s/-I//g)/nanomsg  || echo "")',
+            '<!@(pkg-config libnanomsg --cflags | sed s/-I//g || echo "")',
           ],
           'libraries': [
-            '<!@(pkg-config libnanomsg --libs)',
+            '<!@(pkg-config nanomsg --libs || echo "")',
+            '<!@(pkg-config libnanomsg --libs || echo "")',
+          ],
+        }],
+        ['OS=="mac" and use_system_libnanomsg=="true"', {
+          'include_dirs+': [
+            '<!@(pkg-config libnanomsg --cflags | sed s/-I//g || echo "")',
+            '<!@(pkg-config nanomsg --cflags | sed s/-I//g || echo "")',
+          ],
+          'libraries': [
+            '<!@(pkg-config libnanomsg --libs || echo "")',
+            '<!@(pkg-config nanomsg --libs || echo "")',
           ],
         }],
         ['OS=="win"', {

--- a/src/node_nanomsg.cc
+++ b/src/node_nanomsg.cc
@@ -3,18 +3,18 @@
 #include "node_pointer.h"
 
 #include <nn.h>
-#include <protocol.h>
+
+#include <inproc.h>
+#include <ipc.h>
+#include <tcp.h>
+#include <ws.h>
+
 #include <pubsub.h>
 #include <pipeline.h>
 #include <bus.h>
 #include <pair.h>
 #include <reqrep.h>
 #include <survey.h>
-#include <inproc.h>
-#include <ipc.h>
-#include <tcp.h>
-#include <transport.h>
-#include <ws.h>
 
 using v8::Array;
 using v8::Function;

